lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+class Takeaway
+
+  attr_reader :menu, :order
+
+  def initialize
+    @menu = { Cheeseburger: 5, Fries: 3, Milkshake: 2.99  }
+    @order = Hash.new(0)
+  end
+
+  def show_menu
+    @menu.map do | dish, price |
+     "%s £%.2f" % [dish, price]
+   end.join(", ")
```

Comment:
Lint/BlockAlignment: `end` at 13, 3 is not aligned with `@menu.map do | dish, price |` at 11, 4.